assets/agenda/actions.js

```diff
@@ -675,3 +683,22 @@ export function stopWatchingCoverage(coverage, item) {
             }, (error) => { errorHandler(error, dispatch);});
     };
 }
+
+export function fetchUser(id) {
+    return function (dispatch) {
+        return server.get(`/users/${id}`)
+            .then((data) => {
+                dispatch(getUser(data));
+            })
+            .catch((error) => errorHandler(error, dispatch, setError));
```

Comment:
what I had in mind was handling that 403 server response here and set some flag on state which would be then used in render to display the message, not adding an extra request for the profile